### ISSUE TITLE
A11y: Form fields not programmatically marked as incorrect (typo?)

### ISSUE THREAD

**Issue by blacksheep--** [shopware/platform#7030]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

An invalid=“true” attribute is set on the input fields in the event of an error. This probably refers to an aria-invalid=“true” attribute. As a result of the typo, the screen reader does not read out that the field is filled in incorrectly.

*_Expected behaviour:_*

Instead of invalid=“true”, aria-invalid=“true” should be set via JavaScript.

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag

1. navigate to /account/profile
2. Remove value of one of the inputs (e.g. firstname)
3. Leave the field

![Image](https://github.com/user-attachments/assets/39771c05-dd81-4433-ba5f-4648e204325b)

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40807**



[created from NEXT-40807, comment 541307]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40807, comment 541564]

**Comment by Phil23** [comment]
Hey @blacksheep-- 

The accessibility of form validation was reworked entirely in Shopware 6.6.10.0. Your screenshot shows, that you do not have these changes, yet. I recommend updating to the latest minor version. The 6.6.10.0 will also get extended support, including additional accessibility changes, that might be introduced later.

This is the pull request that introduced the changes: https://github.com/shopware/shopware/pull/5901
